File: xmlIWebServisi/src/app/model/a1/PodaciOAutorimaDTO.ts
import { AutoriDTO } from "./AutoriDTO";
import { TLice } from "../TLice";


export class PodaciOAutorimaDTO{


    constructor(
        public autor:TLice = new TLice(),
        public autori:AutoriDTO = new AutoriDTO(),
        public anonimno:boolean = false
    ){}
}

File: BACKEND/autorska/src/main/java/com/example/autorska/dto/PodaciOAutorimaDTO.java
package com.example.autorska.dto;


import lombok.AllArgsConstructor;
import lombok.NoArgsConstructor;

import java.util.List;

@AllArgsConstructor
@NoArgsConstructor
public class PodaciOAutorimaDTO {

    public TLiceDTO autor;
    public List<AutorDTO> autori;
    public boolean anonimno;

    @Override
    public String toString() {
        return "PodaciOAutorimaDTO{" +
                "autor=" + autor +
                ", autori=" + autori +
                ", anonimno=" + anonimno +
                '}';
    }
}
